src/components/picker/index.js

```diff
@@ -271,6 +273,25 @@ class Picker extends Component {
     return _.get(selectedItem, 'label');
   }
 
+  getFilteredChildren = memoize((children, searchValue) => {
+    const {getItemLabel: getItemLabelPicker} = this.props;
+    return _.filter(children, child => {
+      const {label, value, getItemLabel} = child.props;
+      const itemLabel = getItemLabelPresenter(label, value, getItemLabel || getItemLabelPicker);
+      return !shouldFilterOut(searchValue, itemLabel);
+    });
+  });
+
+  get children() {
+    const {searchValue} = this.state;
+    const {children, showSearch} = this.props;
+    if (showSearch && !_.isEmpty(searchValue)) {
+      return this.getFilteredChildren(children, searchValue);
+    }
+
+    return children;
+  }
+
```

Comment:
Looks good! we should remove the search logic we have in PickerItem code since this one covers it.

Reply:
Done